lib/src/rendering/sliver_sticky_header.dart

```diff
@@ -86,55 +87,53 @@ class RenderSliverStickyHeader extends RenderSliver with RenderSliverHelpers {
   @override
   void attach(PipelineOwner owner) {
     super.attach(owner);
-    if (_header != null) _header.attach(owner);
-    if (_child != null) _child.attach(owner);
+    if (_header != null) _header!.attach(owner);
+    if (_child != null) _child!.attach(owner);
   }
 
   @override
   void detach() {
     super.detach();
-    if (_header != null) _header.detach();
-    if (_child != null) _child.detach();
+    if (_header != null) _header!.detach();
+    if (_child != null) _child!.detach();
   }
 
   @override
   void redepthChildren() {
-    if (_header != null) redepthChild(_header);
-    if (_child != null) redepthChild(_child);
+    if (_header != null) redepthChild(_header!);
+    if (_child != null) redepthChild(_child!);
   }
 
   @override
   void visitChildren(RenderObjectVisitor visitor) {
-    if (_header != null) visitor(_header);
-    if (_child != null) visitor(_child);
+    if (_header != null) visitor(_header!);
+    if (_child != null) visitor(_child!);
   }
 
   @override
   List<DiagnosticsNode> debugDescribeChildren() {
     List<DiagnosticsNode> result = <DiagnosticsNode>[];
     if (header != null) {
-      result.add(header.toDiagnosticsNode(name: 'header'));
+      result.add(header!.toDiagnosticsNode(name: 'header'));
     }
     if (child != null) {
-      result.add(child.toDiagnosticsNode(name: 'child'));
+      result.add(child!.toDiagnosticsNode(name: 'child'));
     }
     return result;
   }
 
-  double computeHeaderExtent() {
+  double? computeHeaderExtent() {
```

Comment:
If I read this correctly, the value cannot be null here, so we could make it a `double `and not a `double?`